egs2/slue-voxpopuli/slu1/local/data.sh

```diff
@@ -35,7 +35,7 @@ fi
 
 if [ ${stage} -le 1 ] && [ ${stop_stage} -ge 1 ]; then
     if [ ! -e "${VOXPOPULI}/LICENSE.txt" ]; then
-	echo "stage 1: Download data to ${VOXPOPULI}"
+	echo "stage 1: Please download data to ${VOXPOPULI}"
```

Comment:
It will be good to also print a link to dataset download instructions.

Reply:
Added in the README